packages/web-components/src/components/promo-item/promo-item.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { customElement, html } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import settings from 'carbon-components/es/globals/js/settings';
+import DDSContentItem from '../content-item/content-item';
+import styles from './promo-item.scss';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Promo item.
+ *
+ * @element dds-promo-item
+ * @slot heading - The Card heading.
+ * @slot image - The Image content.
+ * @slot statistic - The statistic content.
+ * @slot pictogram - The pictogram content.
+ * @slot copy - The Content Item Copy.
+ * @slot footer - The link with icon content.
+ */
+@customElement(`${ddsPrefix}-promo-item`)
+class DDSPromoItem extends StableSelectorMixin(DDSContentItem) {
+  render() {
+    return html`
+      <div class="${prefix}--promo-item__heading">
+        <slot name="heading"></slot>
+      </div>
+      <div class="${prefix}--promo-item__media">
+        <slot class="${prefix}--promo-item__image" name="image" @slotchange="${this._handleSlotChange}"></slot>
+        <slot class="${prefix}--promo-item__statistic" name="statistic" @slotchange="${this._handleSlotChange}"></slot>
+        <slot class="${prefix}--promo-item__pictogram" name="pictogram" @slotchange="${this._handleSlotChange}"></slot>
+      </div>
+      <div class="${prefix}--promo-item__content">
+        <slot class="${prefix}--promo-item__copy" name="copy"></slot>
+      </div>
+      <div class="${prefix}--promo-item__footer">
+        <slot class="${prefix}--promo-item__cta" name="footer"></slot>
+      </div>
+    `;
+  }
+
+  static get stableSelector() {
+    return `${ddsPrefix}--promo-item`;
+  }
+
+  static styles = styles;
+}
+
+export default DDSPromoItem;
```

Comment:
One other thing, since the component needs to be experimental, we need to export the component only if the feature flag is enabled:

```
// Define the new element
if (DDS_PROMO_ITEM) {
  customElements.define(`${ddsPrefix}-promo-item`, DDSPromoItem);
}

export default !DDS_PROMO_ITEM ? undefined : DDSPromoItem;
```

More details (under `Wrapping a Component with a Feature Flag`):
http://ibmdotcom-web-components.mybluemix.net/?path=/docs/overview-feature-flags--page

Reply:
@jeffchew Thanks for pointing this out. I need to remember to wrap the components as well and not just the Storybook stories.

Also, I'm using `DDS_PROMO_GROUP` feature flag on `promo-item.ts` component because I'd like to have a single flag that enables both `promo-group` and `promo-item`. Hope that's okay. https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/5142/commits/3c2ffc55067af0c50b3e6becbb7bf60aa32fdfb8#diff-e5edf0c909ba82c9b6693c1453c2577382ae866bb225dfc861102f59d2e96bb9R60

Happy to adjust it if this doesn't follow your naming conventions or docs.

Reply:
I think the single feature flag should be fine, thank you!